### PR TITLE
chore: update scriptappy

### PR DESCRIPTION
**Pull request by niekvanstaveren**
In preparation of the move to sn-gh-workflows

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,6 +91,8 @@
     "@playwright/test": "1.38.0",
     "@qlik/eslint-config": "0.4.18",
     "@qlik/prettier-config": "0.3.0",
+    "@scriptappy/cli": "^0.10.0",
```
Comment (cbt1) [inline package.json:94]: Pin versions

Comment (niekvanstaveren) [reply package.json:94]: Cannot believe you have to keep pointing this out 😊 
Maybe there is a way to have them pinned when installed actually, I will have a look around. Thanks!

Comment (cbt1) [reply package.json]: Maybe when you install an explicit version they get pinned? But yeah that would be nice to have since it easy to forget

**Review by cbt1 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (niekvanstaveren) [reply package.json]: And of course it exists https://classic.yarnpkg.com/lang/en/docs/cli/add/#toc-yarn-add-exact-e

Comment (niekvanstaveren) [reply package.json]: Even better https://github.com/yarnpkg/yarn/issues/3045

**Pull request merged**